tests/integration/otel/test_utils.py

```diff
@@ -0,0 +1,774 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import json
+import logging
+import pprint
+
+from sqlalchemy import inspect
+
+from airflow.models import Base
+
+log = logging.getLogger("integration.otel.test_utils")
+
+
+def dump_airflow_metadata_db(session):
```

Comment:
Have a look at [tests_common.test_utils.db.py](https://github.com/apache/airflow/blob/main/tests_common/test_utils/db.py), specifically [clear_all()](https://github.com/apache/airflow/blob/main/tests_common/test_utils/db.py#L324) may be handy.